pgmpy/models/MarkovModel.py

```diff
@@ -676,3 +676,30 @@ def get_partition_function(self):
             raise ValueError('Factor for all the random variables not defined.')
 
         return np.sum(factor.values)
+
+    def copy(self):
+        """
+        Returns a deep copy of this Markov Model.
+ 
+        Returns
+        -------
+        MarkovModel: Deep copy of this Markov model.
```

Comment:
Not sure how this is misleading - https://en.wikipedia.org/wiki/Object_copying#Deep_copy

> An alternative is a deep copy, meaning that fields are dereferenced: rather than references to objects being copied, new copy objects are created for any referenced objects, and references to these placed in B.

Deep copy is not specific to Python. The general meaning is what programmers usually think about when they hear deep copy - not `copy.deepcopy`.

However, if you still persist, I can make the change.